xclim/indices/generic.py

```diff
@@ -291,15 +327,19 @@ def count_level_crossings(
     high_data = convert_units_to(high_data, low_data)
     threshold = convert_units_to(threshold, low_data)
 
-    lower = compare(low_data, "<", threshold)
-    higher = compare(high_data, ">=", threshold)
+    lower = compare(low_data, op_low, threshold, constrain=("<", "<="))
+    higher = compare(high_data, op_high, threshold, constrain=(">", ">="))
 
     out = (lower & higher).resample(time=freq).sum()
     return to_agg_units(out, low_data, "count", dim="time")
 
 
 def count_occurrences(
-    data: xr.DataArray, threshold: str, condition: str, freq: str
+    data: xr.DataArray,
+    threshold: str,
+    freq: str,
+    op: str,
```

Comment:
I think the change from "condition" to "op" has to be made in the automatic clix-meta parser in `xclim.core.utils.adapt_clix_meta_yaml` too.

Reply:
I'm trying to generate a YAML file via clix-meta so that I can test these changes, but it's not very clear to me how that's done. Can you explain how, or pass me a YAML file example?

Reply:
You need to install their python package which is in the "python" folder of the directory: https://github.com/clix-meta/clix-meta/tree/main/python
and then run `ci-meta-exporter path/to/the/excel/file.xls`.

Reply:
Fixed. Test for this is now added.